wordpress-mobile/AztecEditor-iOS
# Makes the Example App cartfile private.

### Description:

This PR makes the Example App's Carfile private.  This means dependencies won't see this file, since it doesn't affect any consumable products.

### Testing:

1. Run the following commands from the command line:

```bash
cd Example/
rm -rf Carthage/
carthage update --platform iOS
```

2. Make sure Girdicons is built.
3. Open the project and build the example app.
